extensions/binding/org.eclipse.smarthome.binding.tradfri/ESH-INF/thing/thing-types.xml

```diff
@@ -93,10 +94,7 @@
     <channel-type id="brightness">
         <item-type>Dimmer</item-type>
         <label>Brightness</label>
-        <description>
-            The brightness channel allows to control the brightness of a light.
-            It is also possible to switch the light on and off.
-        </description>
+        <description>Control the brightness and switch the light on and off.</description>
```

Comment:
Did you check the formulations on our other bindings? I wonder if we should somewhere document some guidelines on how labels & descriptions should be written, wdyt?